recipe/patches/0003-fix-lljit-symbols.patch

```diff
@@ -0,0 +1,193 @@
+diff --git a/llvm/include/llvm/ExecutionEngine/Orc/EPCEHFrameRegistrar.h b/llvm/include/llvm/ExecutionEngine/Orc/EPCEHFrameRegistrar.h
```

Comment:
We prefer to have the commit message and metadata in the recipe. To do this, the best way is to take your checkout of llvm, then do:
```
git checkout tags/llvmorg-17.0.2
git checkout -b conda
find ../path/to/llvmdev-feedstock/recipe/patches/ | xargs git am
git cherry-pick 122ebe3b500190b1f408e2e6db753853e297ba28  # the commit you want
git format-patch tags/llvmorg-17.0.2 --no-signature
```

Reply:
Thanks for the instructions!

Step 3 is failing with this:

```console
$ find ../llvmdev-feedstock/recipe/patches/| xargs git am
Applying: pass through QEMU_LD_PREFIX & SDKROOT
Applying: 
fatal: empty ident name (for <>) not allowed
```

Any idea on what is going on?

Reply:
> Any idea on what is going on?

I'm pretty sure it didn't work because of the presence of your `0003-fix-lljit-symbols.patch` in that folder, which git does not know how to apply (with `git am` at least). In any case, I did it for you now. :)